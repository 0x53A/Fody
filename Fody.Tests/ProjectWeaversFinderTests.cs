using System;
using ApprovalTests;
using Moq;
using NUnit.Framework;

[TestFixture]
public class ProjectWeaversFinderTests
{
    [Test]
    public void NotFound()
    {
        var loggerMock = new Mock<BuildLogger>();
        loggerMock.Setup(x => x.LogDebug(It.IsAny<string>()));
        var logger = loggerMock.Object;
        var weavingException = Assert.Throws<WeavingException>(() => ConfigFileFinder.FindWeaverConfigs(Environment.CurrentDirectory, Environment.CurrentDirectory, logger));
        Approvals.Verify(weavingException.Message);
        
    }
}