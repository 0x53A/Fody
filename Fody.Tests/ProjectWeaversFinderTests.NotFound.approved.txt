Could not file a FodyWeavers.xml at the project level (C:\Users\simon\AppData\Local\JetBrains\Installations\ReSharperPlatformVs14\FodyWeavers.xml). Some project types do not support using NuGet to add content files e.g. netstandard projects. In these cases it is necessary to manually add a FodyWeavers.xml to the project. Example content: 
<?xml version="1.0" encoding="utf-8" ?>
<Weavers>
  <WeaverName/>
</Weavers>
